yt_dlp/options.py

```diff
@@ -476,7 +476,8 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
                 'youtube-dl': ['all', '-multistreams', '-playlist-match-filter', '-manifest-filesize-approx'],
                 'youtube-dlc': ['all', '-no-youtube-channel-redirect', '-no-live-chat', '-playlist-match-filter', '-manifest-filesize-approx'],
                 '2021': ['2022', 'no-certifi', 'filename-sanitization', 'no-youtube-prefer-utc-upload-date'],
-                '2022': ['no-external-downloader-progress', 'playlist-match-filter', 'prefer-legacy-http-handler', 'manifest-filesize-approx'],
+                '2022': ['2023', 'no-external-downloader-progress', 'playlist-match-filter', 'prefer-legacy-http-handler', 'manifest-filesize-approx'],
+                '2023': ['prefer-legacy-http-handler'],
```

Comment:
`manifest-filesize-approx` was the default behavior for the majority of 2023 and `prefer-legacy-handler` was being duplicated
```suggestion
                '2022': ['2023', 'no-external-downloader-progress', 'playlist-match-filter'],
                '2023': ['prefer-legacy-http-handler', 'manifest-filesize-approx'],
```